yt/python/yt/wrapper/tests/test_operations.py

```diff
@@ -185,6 +185,11 @@ def test_sort(self):
 
         assert yt.is_sorted(table)
 
+        yt.run_sort(table, sort_by=[SortColumn("x", sort_order="descending")])
+        assert list(reversed([{"x": x, "y": y} for x, y in sorted(columns, key=lambda c: c[0])])) == list(yt.read_table(table))
```

Comment:
`reversed` over `sorted` - a little bit complicated.
may be `sorted(.. reverse=True)`?

Reply:
Thanks, now it looks better